operator/src/test/java/org/keycloak/operator/OperatorE2EIT.java

```diff
@@ -0,0 +1,75 @@
+package org.keycloak.operator;
+
+import io.fabric8.kubernetes.api.model.Node;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import io.quarkus.test.junit.QuarkusTest;
+import org.awaitility.Awaitility;
+import org.awaitility.core.ConditionTimeoutException;
+import org.junit.jupiter.api.Test;
+import org.keycloak.operator.v2alpha1.crds.Keycloak;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+public class OperatorE2EIT extends ClusterOperatorTest {
+
+    @Test
+    public void given_ClusterAndOperatorRunning_when_KeycloakCRCreated_Then_KeycloakStructureIsDeployedAndStatusIsOK() {
+        logger.info(((testremote) ? "Remote " : "Local ") + "Run Test :" + operator + " -- " + namespace);
+
+        // Node
+        List<Node> nodes = k8sclient.nodes().list().getItems();
+        assertThat(nodes).hasSize(1);
```

Comment:
This can be removed as it will limit the execution of the tests on Clusters with just 1 node.

Reply:
ok